cli/src/torii/routing.rs

```diff
@@ -396,10 +384,8 @@ async fn handle_version(sumeragi: Arc<Sumeragi>) -> Json {
     #[allow(clippy::expect_used)]
     let string = sumeragi
         .wsv_mutex_access()
-        .blocks()
-        .last()
+        .latest_block_ref()
```

Comment:
Such granularity is fine here, but in general if you're writing a library (and all crates outside tools and `client_cli` are), the previous syntax is preferable. The compiler is smart enough to treat both appropriately.